source/beamlines.md

```diff
@@ -331,15 +342,54 @@ elements determines the order in which they should tracked through. For example,
 if a line contains the two zero length elements:
 ```{code} yaml
 BeamLine:
-  - kind: Line 
-    name: line
-  - markerA
-  - markerB
+  line:
+    - markerA
+    - markerB
 ```
 then the order of tracking will be `markerA` followed by `markerB`.
 
 %---------------------------------------------------------------------------------------------------
 (s:superposition)=
```

Comment:
Why call this operation a "superposition" as it effectively is an insertion? I would propose to call it "insertion". 
Actually, I see that it is also used for superposition.
To discuss.

Reply:
Discussed.